src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1732,11 +1740,18 @@ class LedgerRPC_test : public beast::unit_test::suite
         testLedgerEntryPayChan();
         testLedgerEntryRippleState();
         testLedgerEntryTicket();
-        testLedgerEntryUnknownOption();
         testLookupLedger();
         testNoQueue();
         testQueue();
         testLedgerAccountsOption();
+
+        // version specific tests
+        for (auto testVersion = RPC::apiMinimumSupportedVersion;
```

Comment:
It could be beneficial to extract this into some sort of helper function (maybe in the next pr) as i imagine this will be copy-pasted all over the place very soon. Better do it sooner than later. I'd go with a variadic template accepting callables (functions expecting api version as their input).

Reply:
you are right, it will get copied. I'll try this out in the next PR. Thank you for pointing it out.

Reply:
any idea what would be the most appropriate file to put this helper in ? `RPCHelpers` ? (cc: @ximinez )

Reply:
Since it's only going to be used in tests, it should be under `src/test`. Env.h already has a lot of stuff, including the functions to make RPC calls, so maybe a static member or standalone function in there... Alternately, just create a new RPCUtils.h under src/test/jtx.

Another alternative if you feel like rewriting this test again is to create a member function of `Env`, so you can loop over each test case individually without the overhead of creating a new `Env` every time.

I don't know for sure, though.